### ISSUE TITLE
Closing Minecraft using ALT + F4 does not end Java

### ISSUE THREAD

**Issue by FooterManDev** [killpowa/Twickt-Launcher#1598]
- [ y ] I searched the issue tracker for other issues covering my case
- [ y ] This is a Launcher issue. (Usually if you can reproduce it with a fresh instance)

**Describe the bug**
When closing Minecraft using ALT + F4, only the window will close, Java will still be running in the background, so you can't launch the game again until you manually end the task.

**To Reproduce**
* Run an instance (In my case, a Forge 1.19.2 instance)
* Close the game using ALT + F4

**Expected behavior**
Java to not be running.

**Possible solutions**
Add some way to close Java when closing the game via ALT + F4.

**Screenshots**

**Operating System:**
- OS: [Linux Mint]
- Launcher Version: [1.1.30]
- Java Version: [17]
- Minecraft Version: [1.19.2]
- Mod Loader: [Forge]

**Additional context**

**Comment by Eskaan** [comment]
Probably a duplicate of #1586 

Edit: We can only try to close the process that we spawned with the game (because we know the pid). If that option isn’t available in the launcher after quitting, this sounds like the process is disowned and should rather be killed using some task manager.
